sources/react/index-footer.jsx

```diff
@@ -18,7 +18,7 @@
 				viewBox="0 0 20 20"
 			>
 				<path d={ path } />
```

Comment:
Was this meant to be updated to `Path` ? As it stands, I don't think `Path` is being used.

Reply:
Good catch.